app/controllers/followers_controller.rb

```diff
@@ -2,13 +2,10 @@ class FollowersController < ApplicationController
 
   # get a followers list for a particle user
   def followers_list
-    ids = User.find_by_id(params[:id]).followers.map { |i| i.follower_id }
-    @users = User.where(id: ids)
+    @users = User.find_by_id(params[:id]).followers
```

Comment:
User.followers / User.followings return records from Followers table, not User table. That's why I used the map to find the followers id and then query them from the User table. I can move the logic to the User model clean this up.